scripts/unix_share_build.sh

```diff
@@ -277,6 +277,17 @@ function mac_build_plugin_pkg(){
     cp /usr/local/opt/szip/lib/libsz.2.dylib .
     install_name_tool -change /usr/local/opt/szip/lib/libsz.2.dylib @rpath/libsz.2.dylib libsvalinn_plugin.so
 
+    # restoring correct RPATH for 17.1 (bin does not exist as it is not shipped with SDK)
+    if [ "$1" == "17.1.0" ] ; then
+        install_name_tool -rpath ${CUBIT_PATH}/bin/plugins/svalinn ${CUBIT_PATH}/MacOS/plugins/svalinn libsvalinn_plugin.so
+        install_name_tool -rpath ${CUBIT_PATH}/bin/plugins/svalinn ${CUBIT_PATH}/MacOS/plugins/svalinn libiGeom.dylib
+        install_name_tool -rpath ${CUBIT_PATH}/bin/plugins/svalinn ${CUBIT_PATH}/MacOS/plugins/svalinn libmcnp2cad.dylib
```

Comment:
Do these put two paths in the path? Do we need both, or just not know which one?

Reply:
this is basically the first one by the second:
```
INSTALL_NAME_TOOL(1)                                                                                      INSTALL_NAME_TOOL(1)

NAME
       install_name_tool - change dynamic shared library install names

SYNOPSIS
       install_name_tool  [-change  old  new  ]  ... [-rpath old new ] ... [-add_rpath new ] ... [-delete_rpath new ] ... [-id
       name] file

[...]
       -rpath old new
              Changes  the  rpath  path name old to new in the specified Mach-O binary.  More than one of these options can be
              specified.  If the Mach-O binary does not contain the old rpath path name in a specified -rpath it is an  error.

```